schemas/payload.js

```diff
@@ -57,6 +57,11 @@ module.exports = {
       "description": "Caches are mounted within the docker container at the mount point specified. Example: ```{ \"CACHE NAME\": \"/mount/path/in/container\" }```",
       "type": "object",
     },
+    "capabilities": {
+      "title": "Capabilities that must be available/enabled for the task container.",
+      "description": "Set of capabilities that must be enabled or made available to the task container Example: ```{ \"capabilities\": { \"privileged\": true }```",
+      "type": "object",
+    },
```

Comment:
Can't we type this some more... Ie. specify all properties, so it's documented what properties are allowed.
You don't have to make them required... ie:
```js
properties: {
  privileged: {
    title: "Privileged Container",
    description: "Run task in a privileged container, equivalent to running docker with `--privileged`, this only works for workerTypes configured to allow this.",
    type: "boolean",
    default: false
  }
},
additionalProperties: false,
default: {} // Specify default value, this very nice for docs, we can also use JSON schema validator
            // that substituted default value in. This is awesome because we don't have to handle the cases
            // in our code... And we don't have to hardcode `false` as default anywhere else.
            // In this case we're unlikely to ever change the default, but might change it for say cpu-shares
            // if we add such a capability later on.
```
If you decide to additionally require a scope this would also be a good place to document it.

Note, I specify `additionalProperties: false`, but I don't not specify `required: ["privileged"]`, because people don't have to specify this property.